packages/ui/src/primitives/tailwind/Modal/index.tsx

```diff
@@ -114,33 +128,39 @@ export const ModalFooter = ({
 const Modal = ({
   title,
   onClose,
+  onCustom,
```

Comment:
we should not put 3 separate props just for this modal

in these cases, it is best to make another modal - also it looks like publish modal will need a modal of its own at some point

i can help create this modal quickly if you want

Reply:
Yes please do